lib/fetch.js

```diff
@@ -100,3 +100,16 @@ export async function fetchNewsArticle(_, args, req) {
 export async function fetchAllNews() {
   return fetchAll('news');
 }
+
+export async function fetchDocsTaggedWith(_, args) {
+  const api = await Prismic.api(PrismicConfig.apiEndpoint);
+  const res = await api.form('everything').ref(api.master()).query(
+    Prismic.Predicates.any('document.tags', [args.tag]))
+    .submit();
```

Comment:
I feel putting the last `)` on the previous line make it look like `submit` is being called on `query`